src/global.d.ts

```diff
@@ -1 +1,24 @@
-declare module 'common-spectrum';
+declare module 'common-spectrum' {
```

Comment:
Please create a PR in common-spectrum so that you can use it everywhere.

Reply:
I'll use it for migration later, apparently I'll need to do it in any case 😫